libsrc/plus4/kgetin.s

```diff
@@ -0,0 +1,14 @@
+.export GETIN
+
+.include "plus4.inc"
+
+KERNAL_GETIN := $FFE4
```

Comment:
Already available here: `master/asminc/cbm_kernal.inc`.

Reply:
Yes, it is.

Reply:
I guess because it should use the definition from cbm_kernal.inc.

Reply:
obviously :)

Reply:
please look at libsrc/c64/kplot.s as an example